server/static/dashboard/replay.js

```diff
@@ -0,0 +1,105 @@
+var connection = new window.sharedb.Connection(new WebSocket(shareURL));
+
+$('#startReplay').click(e => {
+  var targetProject = $('#targetProject')[0].value;
+  var startTimestamp = Date.parse($('#startTimestamp')[0].value);
+  startReplay(startTimestamp, targetProject);
+});
+
+$('#stopReplay').click(e => stopReplay());
+
+var ops = []; // raw ops
+var newcollabids = {}; // old collab id => new collab id
+var collabs = {}; // old collabid => { doc: collabdoc, users: [], files: { path => doc } }
+var replayID;
+
+$.ajax('/ops/' + project).done(function(_ops) {
+  ops = _ops.ops;
+  newcollabids = _ops.newcollabids;
+  console.log('fetched');
+  ops.sort((a, b) => (b.m.ts - a.m.ts) == 0 ? (b.v - a.v) : (b.m.ts - a.m.ts));
```

Comment:
Revisit whether sorting can happen server side. Also: a bug! Need to sort by `v` before `ts` within a single file. Suggests we need a single function to get all ops for a file (as we already have), and then function to get all project ops needs to combine those ops lists without reordering them.